tools/cldr-code/src/main/java/org/unicode/cldr/util/FileReaders.java

```diff
@@ -23,7 +23,22 @@ public static BufferedReader openFile(Class<?> class1, String file) {
         return openFile(class1, file, StandardCharsets.UTF_8);
     }
 
+    /**
+     * @param file path of file relative to class1, or, absolute file if starts with `/`
+     * @return
+     */
     public static BufferedReader openFile(Class<?> class1, String file, Charset charset) {
+        if (charset == null) {
+            charset = StandardCharsets.UTF_8;
+        }
+        if (file.startsWith("/")) {
+            try {
+                return new BufferedReader(
+                        new InputStreamReader(new FileInputStream(file), charset), 1024 * 64);
+            } catch (FileNotFoundException e) {
+                throw new IllegalArgumentException("Could not open file " + file, e);
```

Comment:
Minor, but should probably be UncheckedIOException(...) // I think that's the name
Otherwise looks good.